src/github/api.rs

```diff
@@ -379,6 +381,20 @@ impl Session for GithubSession {
             .map_err(|e| format_err!("error looking up pr labels: {}/{} #{}: {}", owner, repo, number, e))
     }
 
+    fn add_pull_request_labels(&self, owner: &str, repo: &str, number: u32, labels: Vec<String>) -> Result<()> {
+        #[derive(Serialize)]
+        struct AddLabels {
+            labels: Vec<String>,
+        }
+
+        let body = AddLabels {
+            labels: labels,
+        };
+
+        self.client.post_void(&format!("repos/{}/{}/issues/{}/labels", owner, repo, number), &body)
```

Comment:
I tested this out locally and looks like github will create a label for you from this API call if the label doesn't exist yet. :tada: